### PR TITLE
Added Gradle Play Publisher

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
#99

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
@the-dagger the AVD test are failing again rest all is good.

**Comment by championswimmer** [comment]
take version codes from date please

**Comment by championswimmer** [comment]
also set the major and minor versions

**Comment by aggarwalpulkit596** [comment]
@championswimmer done 
i haven't used advanced versioning for versionName, I only used for versionCode.

**Comment by championswimmer** [comment]
Also please add some lines in README.md about how triplet publisher work and which files (yourkeys.json etc) are to be added to make it work and which gradle command to be used etc. So if anyone wants to set up their deployment they would know.

**Comment by aggarwalpulkit596** [comment]
@championswimmer will add that too

File: README.md [hunk 1]
```diff
@@ -26,3 +26,22 @@ It has features such as -
  ## Preview
 
  ![preview](docs/preview.gif)
+ 
+ Automating Publishing to the Play Store
```
Comment (championswimmer) [inline README.md]: make this a level 3 heading

use `###`

File: README.md [hunk 2]
```diff
@@ -26,3 +26,22 @@ It has features such as -
...
+ 
+    -The first APK or App Bundle needs to be uploaded via the Google Play Console because registering the app with the Play Store cannot be done using the Play Developer API.
+    -To use this plugin, you must create a service account with access to the Play Developer API. Once that's done, you'll need to grant the following permissions to your service account for this plugin to work (go to Settings -> Developer account -> API access -> Service Accounts).
+    -Once done download your PKCS12 key or json key somewhere and the location of key in the build.gradle file in the play block
+    -Then run one of the following commands:
+                                        Command                                                  Description
```
Comment (championswimmer) [inline README.md]: use a table format please

File: app/build.gradle [hunk 3]
```diff
@@ -107,3 +123,8 @@ dependencies {
         exclude module: 'design'
     }
 }
+play {
+    jsonFile = file('your-key.json')
+    track = 'beta'
```
Comment (championswimmer) [inline app/build.gradle]: add a comment here that it can be set to production or alpha as well

**Comment by aggarwalpulkit596** [comment]
@championswimmer done

**Pull request merged**